addons/digest/models/digest.py

```diff
@@ -234,7 +234,7 @@ def _compute_tips(self, company, user, tips_count=1, consumed=True):
             '|', ('group_id', 'in', user.groups_id.ids), ('group_id', '=', False)
         ], limit=tips_count)
         tip_descriptions = [
-            self.env['mail.render.mixin']._render_template(tools.html_sanitize(tip.tip_description), 'digest.tip', tip.ids, post_process=True)[tip.id]
+            self.env['mail.render.mixin']._render_template(tools.html_sanitize(tip.tip_description), 'digest.tip', tip.ids, post_process=True, engine="qweb")[tip.id]
```

Comment:
Isn't it possible to make QWeb default rendering engine ? To keep in mind with "small qweb" possible cleaning.

Reply:
Is there any reason to pretend not-qweb is even a template engine at all? Can't it be used just to "render" whatever fields it was created for and that's it?

Reply:
> Is there any reason to pretend not-qweb is even a template engine at all? Can't it be used just to "render" whatever fields it was created for and that's it?

What kind of changes are you suggesting?